src/index.html

```diff
@@ -2,16 +2,176 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      id="form" 
+      method="post"
+    >
+      <fieldset class="fieldset">
+
+        <legend>Personal information</legend>
+
+        <label for="surname" class="label">
+          Surname:
+          <input 
+            type="text"
+            id="surname" 
+            name="surname" 
+            required
+          >
+        </label>
+
+        <label for="name" class="label">
+          Name:
+          <input 
+            type="text" 
+            id="name" 
+            name="name" 
+            required
+          >
+        </label>
+        <label 
+          for="age">
+          How old are You?
+          <input 
+            type="number" 
+            id="age" 
+            name="age" 
+            value="12" 
+            min="1" 
+            max="100"
+          >
+        </label>
+
+        <label for="birthday" class="label">
+          Full date of birth:
+          <input 
+            type="date" 
+            id="birthday" 
+            name="birthday" 
+            required
+          >
+        </label>
+
+        <label for="agreement" class="label">
+          I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            id="agreement" 
+            name="agreement" 
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+
+        <legend>Registration</legend>
+
+        <label for="email" class="label">
+          E-mail:
+          <input 
+            type="email" 
+            id="email" 
+            name="email" 
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label for="password"class="label">
+          Password:
+          <input 
+            type="password" 
+            id="password" 
+            name="password"
+          > 
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+
+        <legend>An interesting fact about you!</legend>
+
+        <label for="cats">
+          Do you love cats?
+          <input type="radio" id="cats" name="cats">
```

Comment:
not critical here, since attributes are really short,
but a common rule is if there are 3 and more attributes - start each one at new line, indented.
```suggestion
          <input
            type="radio"
            id="cats"
            name="cats"
          >
```